### PR TITLE
users should be able to see status of their request

### PR DESCRIPTION
**Pull request by captainamiedi**
#### What does this PR do?

- having a CRUD operations that enable a user see his request status

#### Description of Task to be completed?

- users can see the status of their request

- only users is authorize for these operation

#### How should this be manually tested?

- POSTMAN

#### What are the relevant pivotal tracker stories?

- #167730642

#### Questions:

### CODE REVIEW

File: src/controllers/travelControllers.js [hunk 1]
```diff
@@ -2,8 +2,9 @@ import travelServices from '../services/travelServices';
 import message from '../utils/messageUtils';
 import response from '../utils/response';
 import statusCode from '../utils/statusCode';
+import { async } from 'regenerator-runtime';
```
Comment (chuksjoe) [inline src/controllers/travelControllers.js]: you don't need this line, as it really does nothing

**Review by chuksjoe — CHANGES_REQUESTED** [review]
Good after @captainamiedi, please, why do you have so many commits in your PR?
I will suggest you `squash` them all into one commit so that `rebasing` wouldn't be a headache.

Afterward, you can be updating the commit with `git commit --amend` after staging your changes.

**Comment by Marshalheri** [comment]
Good day team mate.
Please while you work on resolving the conflicts that is present in your work i request that you carefully see what i have done so far based on the changes that we need to implement. 
So that our works can all be in sync with each other.
Thanks

**Review by iakhator — APPROVED** [review]
looks good.

**Pull request merged**